components/layout/Navbar.tsx

```diff
@@ -213,44 +203,51 @@ export function Navbar({ }) {
                                 <Link legacyBehavior href={href}>
                                   <a className="block px-2 py-1.5 text-gray-600 hover:text-blue-500">
                                     {label}{' '}
-                                    {external ? (
+                                    {external && (
                                       <BiLinkExternal
                                         className={`text-blue-200 group-hover:text-blue-400 inline`}
                                       />
-                                    ) : null}
+                                    )}
                                   </a>
                                 </Link>
                               </li>
-                            )
+                            );
                           })}
                       </ul>
                     </li>
-                  )
+                  );
                 }
               })}
             </ul>
             <div className="flex items-center gap-6 min-[1135px]:gap-10">
               <div className="w-full flex justify-start items-center gap-4">
+                <Button color="white" size="small" onClick={openModal}>
+                  <MdEmail className="mr-2"/>
+                  Subscribe
+                </Button>
                 <LinkButton
-                  link="https://app.tina.io/signin"
-                  color="white"
+                  link="https://meetings-eu1.hubspot.com/scott15?uuid=2f12f1a3-8b6a-4472-bfb5-2f3f930a1b19"
```

Comment:
can this _**not**_ be hardcoded

Reply:
as discussed - this will be a different pbi

Reply:
https://github.com/tinacms/tina.io/issues/1822